test/fixtures/v1-vt-05401-state-utility-lowincome.json

```diff
@@ -102,24 +102,23 @@
         "homeowner",
         "renter"
       ],
-      "start_date": "2022",
       "ami_qualification": null,
-      "agi_max_limit": 90001,
+      "agi_max_limit": 90000,
       "filing_status": "joint",
       "eligible": true,
```

Comment:
This is the only new-EV incentive that's eligible, because the filing status and AGI match. There are seven others (four filing statuses x two AGI brackets - this one) that are ineligible.

The diff is confusing because I've excluded the "Replace your ride" incentives, so those are just removed from the fixture. (We don't accurately represent car-scrapping incentives.)